web/src/lib/components/album-page/album-options.svelte

```diff
@@ -80,6 +83,21 @@
       selectedRemoveUser = null;
     }
   };
+
+  const handleSetReadonly = async (user: UserResponseDto, role: AlbumUserRole) => {
```

Comment:
Maybe `handleUpdateSharedUserRole`?

Reply:
Hi @alextran1502 thanks for the suggestion. I have renamed the function, please check once.